### PR TITLE
Update EIP docs for EIP multi NIC

### PR DESCRIPTION
**Pull request by martinkennelly**
Depends on https://github.com/ovn-org/ovn-kubernetes/pull/4040

### CODE REVIEW

File: docs/egress-ip.md [hunk 1]
```diff
@@ -115,27 +115,28 @@ COMMIT
...
 
 No NAT is required on the OVN managed primary network gateway router.
 OVN-Kubernetes (ovnkube-node) takes care of adding a rule to the rule table with src IP of the pod and routed towards a
-new routing table specifically created to route the traffic out the correct interface. IPTables is also altered and an entry
-is created within the chain `OVN-KUBE-EGRESS-IP-Multi-NIC` for each pod to allow SNAT to occur when a src IP is match
-leaving a particular interface.
+new routing table specifically created to route the traffic out the correct interface. IPTables are also altered and an entry
+is created within the chain `OVN-KUBE-EGRESS-IP-Multi-NIC` for each selected pod to allow SNAT to occur when
+egress-ing a particular interface. The routing table number `1111` is generated from the interface name.
+Routes within the main table which egress the interface picked to host the Egress IP are also added to this routing table.
```
Comment (trozet) [inline docs/egress-ip.md:139]: Routes within the main routing table who's output interface share the same interface used for Egress IP are also cloned into the VRF 1111.

File: docs/egress-ip.md [hunk 2]
```diff
@@ -115,27 +115,28 @@ COMMIT
...
+sh-5.2# ip route show table 1111
 default dev dummy
 ```
 
 No NAT is required on the OVN managed primary network gateway router.
 OVN-Kubernetes (ovnkube-node) takes care of adding a rule to the rule table with src IP of the pod and routed towards a
-new routing table specifically created to route the traffic out the correct interface. IPTables is also altered and an entry
-is created within the chain `OVN-KUBE-EGRESS-IP-Multi-NIC` for each pod to allow SNAT to occur when a src IP is match
-leaving a particular interface.
+new routing table specifically created to route the traffic out the correct interface. IPTables are also altered and an entry
```
Comment (trozet) [inline docs/egress-ip.md:136]: IPTables rules are also...

File: docs/egress-ip.md [hunk 3]
```diff
@@ -157,6 +158,24 @@ priority=105,pkt_mark=0x3f0,ip,in_port=2 actions=ct(commit,zone=64000,nat(src=<N
 This is required to make `pod -> node IP` traffic behave the same regardless of where the pod is hosted.  
 Implementation details: https://github.com/ovn-org/ovn-kubernetes/commit/e2c981a42a28e6213d9daf3b4489c18dc2b84b19.
 
+For local gateway mode, in which an Egress IP is assigned to a non-primary interface, an IP rule is added to send packets
+to the main routing table at a priority higher that the EgressIP IP rules, which are set to priority `6000`:
```
Comment (trozet) [inline docs/egress-ip.md:162]: higher than that of

Comment (trozet) [reply docs/egress-ip.md:162]: maybe mention precedence of ip rules is lower number == higher precedence

**Comment by martinkennelly** [comment]
@trozet PTAL

**Review by trozet — APPROVED** [review]

**Pull request merged**